_maps/map_files/Mara17/Mara17_Middle.dmm

```diff
@@ -586,7 +586,7 @@
 /turf/open/floor/deadspace/random/golf_brown,
 /area/mara17/interior/medium/hydrophonics)
 "ce" = (
-/obj/structure/closet/secure_closet/security/mara,
+/obj/structure/closet/secure_closetmara/security,
```

Comment:
Shouldn't it be `/obj/structure/closet/secure_closet/mara/security`?

Reply:
fixed